step.yml

```diff
@@ -6,124 +6,112 @@ website: https://github.com/bitrise-steplib/steps-export-xcarchive
-  - xamarin
+- ios
+- cordova
+- ionic
+- react-native
+- flutter
+- xamarin
 type_tags:
-  - utility
+- utility
 is_always_run: false
 is_skippable: false
 toolkit:
   go:
     package_name: github.com/bitrise-steplib/steps-export-xcarchive
 inputs:
-  - archive_path: $BITRISE_XCARCHIVE_PATH
-    opts:
-      title: iOS or tvOS archive path
-      summary: iOS or tvOS archive path
-      description: |-
-        Path to the iOS or tvOS archive (.xcarchive) which should be exported.
-      is_required: true
-  - export_method: "auto-detect"
-    opts:
-      title: "Select method for export"
-      summary: ""
-      description: |-
-        `auto-detect` option is **DEPRECATED** - use direct export methods!
+- archive_path: $BITRISE_XCARCHIVE_PATH
+  opts:
+    title: iOS or tvOS archive path
+    summary: iOS or tvOS archive path
+    description: |-
+      Path to the iOS or tvOS archive (.xcarchive) which should be exported.
+    is_required: true
+- product: "app"
+  opts:
+    title: "Select a product to distribute"
+    summary: ""
+    description: |-
+      Describes which product to export.
+      Possible options are App or App Clip.
+    value_options:
+    - "app"
+    - "app-clip"
+    is_required: true
+- distribution_method:
+  opts:
+    title: "Distribution method"
+    summary: "Describes how Xcode should export the archive."
+    description: |-
+      Describes how Xcode should export the archive.
+    value_options:
+    - "development"
+    - "app-store"
+    - "ad-hoc"
+    - "enterprise"
+    is_required: true
+- export_development_team:
+  opts:
+    title: "Developer Portal team"
+    summary: "The Developer Portal team to use for this export"
+    description: |-
+      The Developer Portal team to use for this export.
 
-        Describes how Xcode should export the archive.   
-        
-        If you select `auto-detect`, the step will figure out the proper export method  
-        based on the provisioning profile embedded into the generated xcode archive.
-      value_options:
-      - "auto-detect"
-      - "app-store"
-      - "ad-hoc"
-      - "enterprise"
-      - "development"
-      is_required: true
-  - upload_bitcode: "yes"
-    opts:
-      title: "Include bitcode"
-      summary: ""
-      description: |-
-        For __App Store__ exports, should the package include bitcode?
-      value_options:
-      - "yes"
-      - "no"
-      is_required: true
-  - compile_bitcode: "yes"
-    opts:
-      title: "Rebuild from bitcode"
-      summary: ""
-      description: |-
-        For __non-App Store__ exports, should Xcode re-compile the app from bitcode?
-      value_options:
-      - "yes"
-      - "no"
-      is_required: true
-  - team_id: 
-    opts:
-      title: "The Developer Portal team to use for this export"
-      summary: ""
-      description: |-
-        The Developer Portal team to use for this export.
+      Format example: `1MZX23ABCD4`
+    category: "IPA export configuration"
```

Comment:
suggest defining category as a first property, to highlight whether this input is one of the most important ones, or grouped by a category